internal/provider/apppackage_test.go

```diff
@@ -36,7 +36,10 @@ func TestAccResourceAppPackage(t *testing.T) {
 					resource.TestMatchResourceAttr("splunkconfig_app_package.indexes", "files.0.content", regexp.MustCompile("version = 1.0.0")),
 					resource.TestCheckResourceAttr("splunkconfig_app_package.indexes", "files.0.path", "default/app.conf"),
 					resource.TestCheckResourceAttr("splunkconfig_app_package.indexes", "files.1.path", "default/indexes.conf"),
-					resource.TestMatchResourceAttr("splunkconfig_app_package.indexes", "files.1.content", regexp.MustCompile("[original_index]")),
+					resource.TestMatchResourceAttr("splunkconfig_app_package.indexes", "files.1.content", regexp.MustCompile(`\[original_index]`)),
```

Comment:
This regex wasn't actually testing anything close to what we wanted, because the `[` wasn't escaped, thus it was a character class.